src/calculator.js

```diff
@@ -1,17 +1,18 @@
 (function() {
```

Comment:
One of the biggest advantages of browserify is that you do not need function scoping, since that tool does that for you. Please investigate this and remove wrapping functions for all your modules.